development/docker-compose/docker-compose.yaml

```diff
@@ -235,7 +239,7 @@ services:
       identity.masterAdmin.username: admin
       identity.masterAdmin.password: admin
       identity.realmConfiguration.fallbackOptions.enableUser: true
-      identity.realmConfiguration.fallbackOptions.useTemporaryPassword: true
+      identity.realmConfiguration.fallbackOptions.useTemporaryPassword: false
```

Comment:
this is only necessary in case we are using the modelbank realms which have strict password rules, for this local setup is best to keep it as true to make it easier for developers to authenticate

Reply:
In the beginning I thought that the idea is to making this as close as possible to MB. Eventually we will need to disable it when we migrate to MB data since as you mentioned the realms are too strict on password policy. But ATM, I will again enable this flag since you believe it is easier for the developers.